dynamic-layers/neuralnetwork/recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.%.bbappend

```diff
@@ -0,0 +1,4 @@
+do_install:append () {
```

Comment:
I still find this an issue that needs to be fixed in the actual layer. I'm not sure I understand the reasoning not to do that.

Reply:
right this is a generic change.

Reply:
Obviously that is a generic change that is being discussed [upstream](https://github.com/nnstreamer/meta-neural-network/pull/85). The reasoning behind having a bbappend with the fix in our layer is to document the issue in the meantime, while still providing a functional example to the users. I'm not sure if you noticed, but processing with tensorflow did not work until now.

Reply:
I see that the review is in progress, and it introduces some compile issues. Let's see how that settles upstream (see the symlink discussion). I want to avoid:
1. introducing new issues while fixing this
2. making sure that this gets cleaned up after upstream deals with it